style.css

```diff
@@ -96,3 +96,194 @@ main {
 ul li {
   list-style-type: none;
 }
+
+.works {
+  display: grid;
+  width: 100%;
+  background: #f7f7f9;
+  justify-items: center;
+  margin-top: 100px;
+  margin-bottom: 30px;
+  padding-top: 20%;
+  padding-bottom: 10%;
+  row-gap: 100px;
+}
+
+.card1 {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  background-color: #fff;
+  border-radius: 16px;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+}
+
+.card2 {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  background-color: #fff;
+  border-radius: 16px;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+}
+
+.card3 {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  background-color: #fff;
+  border-radius: 16px;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+}
+
+.card4 {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  background-color: #fff;
+  border-radius: 16px;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+}
+
+.works img {
+  height: 100%;
+  width: 100%;
+  left: 16px;
+  top: 16px;
+  border-radius: 8px;
+}
+
+.title {
+  font-family: "Poppins", sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.highlights {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  justify-items: flex-start;
+  padding-left: 0;
+  margin-top: 8px;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  margin-right: 20%;
+}
+
+img.counter {
+  height: 8px;
+  width: 8px;
+  left: 67px;
+  top: 12px;
+  border-radius: 0;
+}
+
+.description {
+  height: 76px;
+  width: 295px;
+  left: 0;
+  top: 92px;
+  border-radius: nullpx;
+}
+
+.languages {
+  display: flex;
+  flex-direction: row;
+  padding-left: 0%;
+  width: 80%;
+}
+
+.stack {
+  background: #ebebff;
+  color: #396df2;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  margin-right: 1rem;
+  letter-spacing: 0.03em;
+  padding: 0.4rem;
+  border-radius: 10px;
+}
+
+.btn {
+  padding: 1em;
+  border: 1px solid #396df2;
+  border-radius: 10px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
```

Comment:
- [ ] Great job with your styling 👍 . Kindly make sure you give all `buttons` interactions as seen in the Figma design 🙏 

![image](https://user-images.githubusercontent.com/74814780/191958421-30522290-2999-4683-9ba0-ea064b731c6c.png)